libdrizzle/binlog.cc

```diff
@@ -337,8 +337,8 @@ drizzle_return_t drizzle_state_binlog_read(drizzle_st *con)
     binlog_event->raw_length= binlog_event->length= drizzle_get_byte4(con->buffer_ptr + 9);
     if (con->packet_size != binlog_event->length)
     {
-        drizzle_set_error(con, __func__,
-                          "packet size error:%" PRIu32 ":%" PRIu32, con->packet_size, binlog_event->length);
+        drizzle_set_error(con, __FILE_LINE_FUNC__, "packet size error:%" PRIu32 \
```

Comment:
I might be wrong here but is the use of `\` to indicate multiple lines inconsistent with how this is done in other modules?